web/basics/binding_to_a_map/my_element.dart

```diff
@@ -2,21 +2,19 @@
 library web.basics.binding_to_a_map;
 
 import 'package:polymer/polymer.dart';
+import 'package:web_components/web_components.dart' show HtmlImport;
+import 'dart:js';
 
-@CustomTag('my-element')
+@PolymerRegister('my-element')
 class MyElement extends PolymerElement {
-  @observable Map person;
-
+  @property
+  JsObject person;
```

Comment:
Hello @jakemac53 , unfortunately `Map` did not work for me here.  I set the `person` property to and it did not work.:

```dart
@property
Map person;
...
person = {
      'name': {'first': 'Kathy', 'last': 'Walrath'},
      'title': 'Writer Extraordinaire',
      'company': {'name': 'Google'}
    };
```